src/views/admin/PendingCredits.vue

```diff
@@ -88,6 +97,15 @@
           showConfirmButton: false,
           timer: 3000
         })
+      },
+      async updatePendingCreditRequests() {
+        try {
+          await this.$store.dispatch('fetchPendingCreditRequests')
+          this.hasLoadingError = false
+        } catch (e:any) {
+          console.error('Failed to fetch pending credit requests', e)
+          this.hasLoadingError = true
+        }
```

Comment:
There are no loading symbol displayed nor managed, is that intended ?

Reply:
This PR does not introduce loading indicators, which was work done by seddik. He did not implement it.
Please see PR #156

Reply:
ok, thanks for the heads up.